src/components/commentCount.stories.tsx

```diff
@@ -1,18 +1,19 @@
 // ----- Imports ----- //
 
 import React, { FC } from 'react';
-import { withKnobs, number, boolean } from '@storybook/addon-knobs';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
 
 import CommentCount from './commentCount';
 import { Pillar, Design, Display } from 'format';
 import { selectPillar } from 'storybookHelpers';
+import { some } from 'types/option';
 
 
 // ----- Stories ----- //
 
 const Default: FC = () =>
     <CommentCount
-        count={number('Count', 1234, { min: 0 })}
+        count={some(1234)}
```

Comment:
Does this drop the control for choosing the number of comments? Could that be useful for testing UIs against small or large comment counts?

Reply:
Yeah it might be nice to keep, did you come across a way of changing optional values?

Reply:
🙂

```tsx
<CommentCount
  count={some(number('Count', 1234, { min: 0 }))}
/>
```

Reply:
https://github.com/guardian/apps-rendering/pull/473